### PR TITLE
Image refresh for fedora-testing

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for fedora-testing
 * [x] image-refresh fedora-testing

### CODE REVIEW

**Comment by jelly** [comment]
Both tests failed with:

```Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/verify/check-system-info", line 821, in testOverview
    wait(lambda: progressValue(1) < 20)
  File "/work/bots/make-checkout-workdir/test/common/testlib.py", line 2339, in wait
    raise Error(msg or "Condition did not become true.")
testlib.Error: Condition did not become true.
```

So that doesn't seem like a flake.

**Comment by jelly** [comment]
This isn't a flake `pmie.service` uses a ton of CPU

image

So maybe we mask it and unmask?

**Comment by martinpitt** [comment]
The [damage is done now](https://bodhi.fedoraproject.org/updates/FEDORA-2022-18bbe14f70), pcp went into stable and can't be -1'ed any more. So either this was a temporary glitch (unlikely, though), or it will now break the next fedora-37 image refresh.

**Comment by jelly** [comment]
> The [damage is done now](https://bodhi.fedoraproject.org/updates/FEDORA-2022-18bbe14f70), pcp went into stable and can't be -1'ed any more. So either this was a temporary glitch (unlikely, though), or it will now break the next fedora-37 image refresh.

Yes, sorry for not following up quickly enough. I'll report a bugzilla

File: naughty/fedora-36/4031-pmie-check-cpu-usage [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "/work/bots/make-checkout-workdir/test/verify/check-system-info", line *, in testOverview
+    wait(lambda: progressValue(1) < 20)
```
Comment (marusak) [inline naughty/fedora-36/4031-pmie-check-cpu-usage:3]: This is bit too generic as there might be a lot of reasons why the load does not come under 20.  I would prefer if we would just disable `pmie.service`. Or is there some reason why we cannot do that?

Comment (marusak) [reply naughty/fedora-36/4031-pmie-check-cpu-usage:3]: right, I was thinking about editing of the test. Good point about disabling pcp service in metrics test but this naughty does not say anything about what consumes the processor. If there is another service in the meantime we will still be blocking this one naughty and the test might never run to completion.

Comment (jelly) [reply naughty/fedora-36/4031-pmie-check-cpu-usage:3]: So instead of checking the overall CPU, check the table of processes?

Comment (jelly) [reply naughty/fedora-36/4031-pmie-check-cpu-usage:3]: Ok, created the workaround here https://github.com/cockpit-project/cockpit/pull/17892

**Review by jelly — APPROVED** [review]

**Pull request merged**